runtime/oti/j9accessbarrier.h

```diff
@@ -165,21 +196,10 @@ typedef struct J9IndexableObject* mm_j9array_t;
 #define J9JAVAARRAYCONTIGUOUS_EA_VM(javaVM, array, index, elemType) \
 	J9JAVAARRAYCONTIGUOUS_BASE_EA_VM(javaVM, array, index, elemType)
 
-#endif /* defined(J9VM_ENV_DATA64) */
-
-#define J9ISCONTIGUOUSARRAY(vmThread, array) \
-	(J9VMTHREAD_COMPRESS_OBJECT_REFERENCES(vmThread) \
-		? (0 != ((J9IndexableObjectContiguousCompressed *)(array))->size) \
-		: (0 != ((J9IndexableObjectContiguousFull *)(array))->size))
-
-#define J9ISCONTIGUOUSARRAY_VM(javaVM, array) \
-	(J9JAVAVM_COMPRESS_OBJECT_REFERENCES(javaVM) \
-		? (0 != ((J9IndexableObjectContiguousCompressed *)(array))->size) \
-		: (0 != ((J9IndexableObjectContiguousFull *)(array))->size))
-
 /* TODO: queries compressed twice - optimize? */
 #define J9JAVAARRAY_EA(vmThread, array, index, elemType) (J9ISCONTIGUOUSARRAY(vmThread, array) ? J9JAVAARRAYCONTIGUOUS_EA(vmThread, array, index, elemType) : J9JAVAARRAYDISCONTIGUOUS_EA(vmThread, array, index, elemType))
```

Comment:
as we discussed in person, we could optimized 32 bit similarly, although there are only 2 layouts (with and without arraylet and no dataAddr), 

but let's do it in a separate change